packages/luvi/luajit-c-bytecode.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/cmake/Modules/LuaJITAddExecutable.cmake b/cmake/Modules/LuaJITAddExecutable.cmake
+index e250d08..22d53ae 100644
+--- a/cmake/Modules/LuaJITAddExecutable.cmake
++++ b/cmake/Modules/LuaJITAddExecutable.cmake
+@@ -24,7 +24,7 @@ MACRO(LUAJIT_add_custom_commands luajit_target)
+       math(EXPR _stripped_file_length "${_luajit_file_length} - ${_luajit_source_dir_length} - 1")
+       string(SUBSTRING ${file} ${_begin} ${_stripped_file_length} stripped_file)
+ 
+-      set(generated_file "${CMAKE_BINARY_DIR}/jitted_tmp/${stripped_file}_${luajit_target}_generated${CMAKE_C_OUTPUT_EXTENSION}")
++      set(generated_file "${CMAKE_BINARY_DIR}/jitted_tmp/${stripped_file}_${luajit_target}_generated.c")
+ 
+       # Only use LUA_PATH if it's set
+       IF(LUA_PATH)
+@@ -67,5 +67,6 @@ ENDMACRO()
+ 
+ MACRO(LUA_ADD_EXECUTABLE luajit_target)
+   LUAJIT_add_custom_commands(${luajit_target} ${ARGN})
++  message("-- target: ${target_srcs}")
```

Comment:
Is this line necessary?
Or this patch in general?
It seems like setting `-DCMAKE_C_OUTPUT_EXTENSION=.c` should do the trick

Reply:
Oops, that was for testing the `re.lua` issue.

Reply:
Resolved in [bd96230](https://github.com/termux/termux-packages/pull/20462/commits/bd962300a58826a5dab430ed70eab7a9bcf0b796).